modules/gapi/include/opencv2/gapi/s11n.hpp

```diff
@@ -229,20 +334,27 @@ IIStream& operator>> (IIStream& is, std::vector<T> &ts) {
 
 // Generic: variant serialization
 namespace detail {
```

Comment:
Can you ban this namespace instead? See `Doxyfile.in`, `EXCLUDE_SYMBOLS`.

Reply:
You are right, removed the `private`s